_episodes/interviews/2024-05-19-glen-de-cauwsemaecker.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Rama with Glen De Cauwsemaecker"
+date: 2024-05-19T10:53:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2024-05-19-glen-de-cauwsemaecker.mp3
+duration: "49:27"
+length: "71675212"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Allen Wyma talks with [Glen De Cauwsemaecker](https://www.glendc.com/) about [Rama](https://ramaproxy.org/), a modular and customizable proxy built in Rust.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+⁃ [@00:00] - Meet Glen De Cauwsemaecker, creator of Rama, a modular service framework in Rust for network packet transformation
```

Comment:
Feels weird to have such a long chapter mark that also includes "new content". Can the Rama description be made part of the episode description up top (fine to have more than one sentence there), and then this shortened to "Introducing Glen"?